srcCppCommon/ParameterManager.cxx

```diff
@@ -1225,7 +1225,43 @@ void ParameterManager::initialize()
             | Middleware::RTIDDSMICRO);
     secureDebug->set_internal(true);
     create("secureDebug", secureDebug);
-#endif
+  #endif
+
+  #ifdef RTI_MICRO
+        Parameter<int> *crc = new Parameter<int>(32);
+        crc->set_command_line_argument("-crc", "<length>");
+        crc->set_description(
+            "Calculate and check CRC checksum for that specific length. 16, 32, 64 or 128");
+        crc->set_type(T_NUMERIC_D);
+        crc->set_extra_argument(YES);
+        crc->set_range(0, 128);
+        crc->set_group(TRANSPORT);
+        crc->set_supported_middleware(
+                Middleware::RTIDDSMICRO);
+        create("crc", crc);
+
+        Parameter<bool> *customCrc = new Parameter<bool>(false);
```

Comment:
Potentially we could get rid of this option if we don't expose custom CRCs to the user. @salvacorts, is it worth exposing it? (See my other comments about this).